typescript/__tests__/testProgramaticallyCreateConfig.ts

```diff
@@ -18,3 +20,69 @@ describe("test extract_override_settings", () => {
     expect(globalSettingsForTestModel).toEqual({ topP: 0.9 });
   });
 });
+
+describe("test extract_override_settings", () => {
```

Comment:
Update `describe and `test` as I explained in the diff below

Reply:
Updated. Please take another look

Reply:
Looks good, ship it